workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,194 @@
+| __Heatmap Chart__  | ---------- |
+| __Hexagon Chart__  | Displays host and container metrics as a function of color  |
+
+### Create Your Query
+
+Create your query logic in the Metrics Section. Select the specific metric you would like to query, aggregate, and filter. Specify the interval at which data is ingested, create metric aliases to easily interpret what is displayed within the widget, and filter out unwanted data within your query.
+
+Think of your queries like 2-dimensional arrays, where __Aggregate By__ is a function that performs a transformation on each subarray and __Group By__ performs an aggregate function on the aggregate outputs across all arrays. 
+
+![](https://lh7-us.googleusercontent.com/8sltD2VI-oNKeR0PPF2cp1rx5ktjK-KfXnF-vOUzvfXbcDElFeTKEXmsx6wm3PDN95xl1IjbcOwyX9Oq5CXoKVcjroG1V0eq3JeaL0XP8cvUU4yqIKKyFQa2t7dDrpSJJKaM6emthaYHDvGNSr3ztOs)
+
+
+#### Group By
+
+The Group By field is the outer most aggregation made in your query. Metric aggregations are contingent on the initial metric selected for your widget. Group your initial metric with resource attributes, metric attributes, or another metric based on the selected metric. 
+
+The below example tracks CPU utilization by host; it is grouping the `system.cpu.utilization` metric by all unique instances of the `host.name` metric:
+
+
+![](https://lh7-us.googleusercontent.com/ecBBt0x3KYh_2QyUksFy16HIKqlpl3k4PUyxIDjgrHD6M1iNkOiS8iR0P7vkHqktOCj8IdhZf-N8o6falplfl8KtNlHE4mFZFr65O_4ibg0OjXrxR3lL5zSPzLhKWppc_2YzQ6T_6ukb8VXkdA4EaYU)
+
+The following table displays all available methods for aggregating your chosen metrics:
+
+| __Aggregation Type__ | __Description__                                                |
+| :------------------- | :------------------------------------------------------------ |
+| __Average By__       | Average of all values in selected time range for given metric |
+| __Sum By__           | Sum of all values in selected time range for given metric |
+| __Minimum By__       | Minimum of all values in selected time range for given metric |
+| __Maximum By__       | Maximum of all values in selected time range for given metric |
+| __Any By__           | Gathers all possible values in selected time range for given metric __and/or__ attribute |
+| __Unique By__        | Count of unique values in selected time range for given metric __and/or__ attribute |
+| __Count By__         | Count of all values in selected time range for given metric __and/or__ attribute |
+| __Aggregated By__    | Groups all values in selected timerange for given metric __and/or__ attribute |
+
+#### Aggregate By
+
+The Aggregate By field is a nested aggregation technique that allows you to aggregate your initial query, the inner layer of data. The below example is grouping the `container.memory.percent` metric by averaging the values of the `container.name` attributes and summing `container.memory.percent`:
```

Comment:
If we change the paragraph for the picture we can then change this wording to link the two. "As described in the above diagram, the Aggregate By field is a nested...."